Tools/flightgear_bridge/src/fg_communicator.cpp

```diff
@@ -0,0 +1,150 @@
+#include "fg_communicator.h"
+
+
+FGCommunicator::FGCommunicator(VehicleState * v)
```

Comment:
You could run the c++ files through the code formatter.

``` Console
$ ./Tools/astyle/fix_code_style.sh Tools/flightgear_bridge/src/fg_communicator.cpp
```

Reply:
done in 09a69af, Thanks for the fix_code_style script link.